src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    static final int iDateStr = 0;
+    static final int iEmployeeName = 1;
```

Comment:
What 'i' means in name of variable? Better to avoid abbreviation if it is not common